TESTING.md

```diff
@@ -169,8 +169,112 @@ cargo make \
   test
```

Comment:
I think the two commits can be squashed in one, since they are both related to the same change.